src/App.js

```diff
@@ -12,6 +13,17 @@ import Footer from 'components/PageSections/Footer';
 
 import 'css/coa.css';
 
+
+const ScrollToTop = () => {
+  const { pathname } = useLocation();
+
+  useEffect(() => {
+    window.scrollTo(0, 0);
```

Comment:
The bugs we'd run into in the past with "window" were with Janis builds. If you've got a root-level window in some function called by react-static, it'll freak out. Because "react-static build" is a server-side build process, and there is only a "window" object on the browser client. But since your window is safely tucked within a useEffect(), I suspect we'll be good. This is a great solution!